builtin/providers/aws/resource_aws_iam_access_key.go

```diff
@@ -115,3 +126,19 @@ func resourceAwsIamAccessKeyDelete(d *schema.ResourceData, meta interface{}) err
 	}
 	return nil
 }
+
+func sesSmtpPasswordFromSecretKey(key *string) string {
+	if key == nil {
+		return ""
+	}
+	version := byte(0x02)
```

Comment:
I don't think the byte cast is necessary, but if it helps for clarity purposes then :+1: